app/services/support_interface/send_ucas_match_initial_emails_multiple_acceptances.rb

```diff
@@ -0,0 +1,17 @@
+module SupportInterface
+  class SendUCASMatchInitialEmailsMultipleAcceptances
+    attr_reader :ucas_match
+
+    def initialize(ucas_match)
+      @ucas_match = ucas_match
+    end
+
+    def call
+      raise 'UCAS Match initial emails already sent' if ucas_match.initial_emails_sent?
+
+      application_choice = ucas_match.application_choices.first
```

Comment:
I think this may be a problem actually. If candidate applied on both services but not for the same course `ucas_match.application_choices` will be nil.

For the multiple acceptances email we don't really care about course/application choice details. We don't include this info in the email. If we want to pass `application_choice` for consistency maybe we could `ucas_match.candidate.application_form.application_choices.first`. 
Although, I would be tempted to just pass `application_form` (we do it in the mailer already). I'm worried that someone (myself in the future 😂 ) can incorrectly assume that we are passing the application from which has an acceptance.